pkg/tests/apis/dashboard/integration/api_validation_test.go

```diff
@@ -113,45 +125,116 @@ func TestIntegrationValidation(t *testing.T) {
 	}
 }
 
-func testIntegrationValidationForServer(t *testing.T, helper *apis.K8sTestHelper, dualWriterMode rest.DualWriterMode) {
-	t.Cleanup(func() {
-		helper.Shutdown()
-	})
+// TestIntegrationDashboardAPIAuthorization tests the dashboard K8s API with authorization checks
+func TestIntegrationDashboardAPIAuthorization(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
 
-	// Create test contexts organization
-	org1Ctx := createTestContext(t, helper, helper.Org1, dualWriterMode)
-	org2Ctx := createTestContext(t, helper, helper.OrgB, dualWriterMode)
+	// TODO: rest.Mode3 is failing on dashboard creation, so we skip it for now
```

Comment:
Is this todo still relevant? The PR description has this crossed out: 2. Mode3 experiences an issue where dashboards in the root folder can not be created. We need to fix this!